src/options.c

```diff
@@ -255,16 +261,12 @@ static void optionsParseLine(char *optarg)
             }
             break;
         case Width:
-            if (!optionsParseIsString(value)) {
-                errx(EXIT_FAILURE, "Missing value for suboption '%s'",
-                    token[Width]);
-            }
-
-            opt.lineWidth = optionsParseRequiredNumber(value);
-
-            if (opt.lineWidth <= 0 || opt.lineWidth > 8) {
-                errx(EXIT_FAILURE, "Value of the range (1..8) for "
-                    "suboption '%s': %d", token[Width], opt.lineWidth);
+            opt.lineWidth = options_parsenum(value, 1, 8, &errmsg);
+            if (errmsg) {
+                if (value == NULL)
```

Comment:
When `--line width` then `value == NULL`

When `--line width=` then `value[0] = '\0'`

Two possible solutions:
 ```c
if (value == NULL || value[0] == '\0')
```

```c
if (!optionsParseIsString(value))
```

Reply:
`options_parse()` handles that
![2023-01-06-145454_624x164_scrot](https://user-images.githubusercontent.com/25590950/211037226-1a6b7b5a-432b-4b32-9025-4bcb5bea204f.png)